src/main/java/org/lisoft/lsml/model/item/AmmoWeapon.java

```diff
@@ -81,19 +90,49 @@ public Ammunition getAmmoType() {
         return ammoType;
     }
 
+    public int getVolleySize() {
```

Comment:
```suggestion
    protected int getVolleySize() {
```
I don't believe that the user of this class would ever be interested in querying the volley parameters directly. Let's keep the class API as small as possible.

Reply:
sounds good to me.

Reply:
done